jobs/transforms/story_validation.py

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Beam DoFns and PTransforms to provide validation of story models."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.domain import story_domain
+from core.platform import models
+from jobs.decorators import validation_decorators
+from jobs.transforms import base_validation
+
+(story_models,) = models.Registry.import_models([models.NAMES.story])
+
+
+@validation_decorators.AuditsExisting(
+    story_models.StorySnapshotMetadataModel,
+    story_models.StoryCommitLogEntryModel)
+class ValidateStoryCommitCmdsSchema(
+        base_validation.BaseValidateCommitCmdsSchema):
+    """Overrides _get_change_domain_class for story models."""
+
+    def _get_change_domain_class(self, input_model): # pylint: disable=unused-argument
```

Comment:
Btw if I look at the original implementation https://github.com/oppia/oppia/blob/7964a49d527164a567c51021b9757bca60b7e683/core/domain/story_validators.py#L127 there seems to be more logic involved?

Reply:
Please also check other places, even the ones that were mentioned before.

Reply:
okay, I see what I missed, I will need to make lots of changes, so I will close this PR for now and open a pr for every 2 models. Is that ok ? or should I update in this PR itself ?
question: 
`StorySnapshotMetadataModelValidator` implements `_get_change_domain_class` as 
`return story_domain.StoryChange` 
while
`StoryCommitLogEntryModelValidator` does it as
`if item.id.startswith('story'):
            return story_domain.StoryChange
        else:
            cls._add_error('error')`

in my `story_validation.py`, I will have to implement it like in `StoryCommitLogEntryModelValidator` right ?